br/pkg/task/common.go

```diff
@@ -310,6 +311,14 @@ func DefineFilterFlags(command *cobra.Command, defaultFilter []string, setHidden
 	}
 }
 
+func DefineWithSysFlags(command *cobra.Command, setHidden bool) {
```

Comment:
this function can be move into `DefineRestoreCommonFlags`

Reply:
[fix comments](https://github.com/pingcap/tidb/pull/36652/commits/1e976ab64458e936afdff84d375a5a7c1b049a53)

Reply:
it seems `--with-sys-table` should be a hidden flag